adaptdl_cli/adaptdl_cli/registry.py

```diff
@@ -75,15 +106,13 @@ def _fix_daemon_json():
 
 
 def registry_running():
-    svc = json.loads(subprocess.check_output(['kubectl', 'get', 'svc', '-l',
-                                             'app=docker-registry', '-ojson']))
-    return len(svc['items']) > 0
+    v1 = kubernetes.client.CoreV1Api()
+    svc = v1.list_service_for_all_namespaces(
+        label_selector="app=docker-registry,release=adaptdl")
```

Comment:
I don't think this hard-coded selector works in general since users can deploy AdaptDL under a different release name, for example if they install the chart using `-g` instead of exactly specifying the release name as `adaptdl`

Reply:
We assign a fixed name `adaptdl-registry` to the service. So `field_selector` should work in any deployment scenario.

Reply:
SGTM. Can you also add a comment to `fullnameOverride` in `values.yaml` saying it should not be changed and why?

Reply:
Added the comment but I think it's kind of implied since someone used name override in the first place.